src/sentry/static/sentry/app/components/events/interfaces/frame/frame.tsx

```diff
@@ -169,7 +194,7 @@ export class Frame extends React.Component {
     const {symbolicatorStatus} = this.props.data;
     const func = this.props.data.function || '<unknown>';
     // Custom color used to match adjacent text.
-    const warningIcon = <IconQuestion size="xs" color="#2c45a8" />;
+    const warningIcon = <IconQuestion size="xs" color={'#2c45a8' as any} />;
```

Comment:
Maybe you could use some color of our theme?

Reply:
@priscilawebdev I thought about that but then I saw the comment about this being a custom color to match the adjacent text and I didn't want to cause a mismatch.

Reply:
Can we add a `TODO(color)` here?